packages/gatsby-recipes/README.md

```diff
@@ -133,56 +137,79 @@ Installs a Gatsby Plugin in the site's `gatsby-config.js`.
 />
 ```
 
-##### props
+#### props
 
 - **name** name of the plugin
 - **options** object with options to be added to the plugin declaration in `gatsby-config.js`. JavaScript code is not _yet_ supported in options e.g. `process.env.API_TOKEN`. This is being worked on. For now only simple values like strings and numbers are supported.
 - **key** string used to distinguish between multiple plugin instances
 
-#### `<GatsbyShadowFile>`
+### `<GatsbyShadowFile>`
 
 ```jsx
 <GatsbyShadowFile theme="gatsby-theme-blog" path="src/components/seo.js" />
 ```
 
-##### props
+#### props
 
 - **theme** the name of the theme (or plugin) which provides the file you'd like to shadow
 - **path** the path to the file within the theme. E.g. the example file above lives at `node_modules/gatsby-theme-blog/src/components/seo.js`
 
-#### `<NPMPackage>`
+### `<NPMPackage>`
 
 `<NPMPackage name="lodash" version="latest" />`
 
-##### props
+#### props
 
 - **name**: name of the package(s) to install. Takes a string or an array of strings.
 - **version**: defaults to latest
 - **dependencyType**: defaults to `production`. Other options include `development`
 
-#### `<NPMScript>`
+### `<NPMScript>`
 
 `<NPMScript name="test" command="jest" />`
 
-##### props
+#### props
 
 - **name:** name of the command
 - **command** the command that's run when the script is called
 
-#### `<File>`
+### `<File>`
 
 <File path="test.md" content="https://raw.githubusercontent.com/KyleAMathews/test-recipes/master/gatsby-recipe-jest.mdx" />
 
-##### props
+#### props
 
 - **path** path to the file that should be created. The path is local to the root of the Node.js project (where the package.json is)
 - **content** URL to the content that should be written to the path. Eventually we'll support directly putting content here after some fixes to MDX.
 
 > Note that this content is stored in a [GitHub gist](https://gist.github.com/). When linking to a gist you'll want to click on the "Raw" button and copy the URL from that page.
 
-## Resources
+## How to setup development environment to work on Gatsby Recipes core
 
-- A free 6 min eggheadio [collection](https://egghead.io/playlists/getting-started-with-gatsbyjs-recipes-c79a) by [Khaled Garbaya](https://twitter.com/khaled_garbaya)
+First [follow the instructions on setting up a local Gatsby dev environment](https://www.gatsbyjs.org/contributing/setting-up-your-local-dev-environment/).
```

Comment:
This is a bit confusing, first implies there are more steps?

Reply:
Moved it around so it made more sense